security/ldap.rst

```diff
@@ -9,7 +9,7 @@ Symfony provides different means to work with an LDAP server.
 The Security component offers:
 
 * The ``ldap`` :doc:`user provider</security/user_provider>`, using the
-  :class:`Symfony\\Component\\Security\\Core\\User\\LdapUserProvider`
+  :class:`Symfony\\Component\\Ldap\\Security\\LdapUserProvider`
```

Comment:
Please remove the spaces at the beginning like before

Reply:
Which spaces? Only the classname has been updated in this PR.

Reply:
Weird on a phone it looks like there were spaces at the beginning. Sorry